utils/src/main/scala/com/salesforce/op/utils/spark/OpSparkListener.scala

```diff
@@ -143,6 +145,11 @@ class OpSparkListener
 
 }
 
+object OpSparkListener {
+  private val SPARK_JOB_GROUP_ID = "spark.jobGroup.id"
+  val DEFAULT_GROUP_ID: String = "other"
```

Comment:
Why not add this to `OpStep` enum? e.g. `case object Other(name: String = "other") extends OpStep(name)`

Reply:
Makes sense, done. It did force me to move `OpStep` from `core` to the `utils` package.